test/kpod.bats

```diff
@@ -2,8 +2,48 @@
 
 load helpers
 
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
```

Comment:
can you drop this and see if Travis is happy? maybe your laptop has some misconfigurations we'll take care of separately from this PR